osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -200,7 +199,7 @@ protected void AddJudgement(Judgement judgement)
         /// </summary>
         /// <param name="userTriggered">Whether the user triggered this process.</param>
         /// <returns>Whether a judgement has occurred from this <see cref="DrawableHitObject"/> or any nested <see cref="DrawableHitObject"/>s.</returns>
-        protected internal sealed override bool UpdateJudgement(bool userTriggered)
+        protected internal bool UpdateJudgement(bool userTriggered)

```

Comment:
I believe it should be possible to remove `internal` from this now?

Reply:
Sure. Was there a reason for it to be there before that is now gone?

Reply:
Because it was implemented in the overridden class, but since this is now implemented in the base class it's fine.